tournament/tournament-api-documentation.md

```diff
@@ -6,17 +6,61 @@
 
 ### Create / retrieve tournament
 
+Retrieve list of available public tournament
+
 <details>
  <summary><code>GET</code> <code><b>/tournament</b></code></summary>
 
-#### Retrieve list of available public tournament
-
 </details>
 
+Create a new tournament
+
 <details>
  <summary><code>POST</code> <code><b>/tournament</b></code></summary>
 
-#### Create a new tournament
+
+### Parameters
+
+#### Body
+
+- Tournament name must be between 3 and 20 characters and can only contain alnum and space
+- Max players must be between 2 and 16 (optional, default = 16 players)
```

Comment:
The word 'max' should be removed

Reply:
67d148d102712042278a9d096d7ad0854cabe7aa